src/dump/dump.c

```diff
@@ -109,9 +109,6 @@ void dump_write_score(FILE *f, gregorio_score *score)
     if (score->score_copyright) {
         fprintf(f, "   score_copyright           %s\n", score->score_copyright);
     }
-    if (score->language) {
-        fprintf(f, "   language                  %s\n", score->language);
-    }
```

Comment:
Yes.  I didn't realize this field wasn't needed after parsing until I refactored the way headers are parsed in making the changes for this pull request.